cmd/generate-plugin-overview/generate-plugin-overview.go

```diff
@@ -32,13 +33,14 @@ const (
 	separator  = " | "
 	pageHeader = `## Available kubectl plugins
 
-The following kubectl plugins are currently available
-on [Krew plugin index](https://sigs.k8s.io/krew-index).
+The following kubectl plugins are available on
+[Krew plugin index](https://sigs.k8s.io/krew-index) as of revision %s.
```

Comment:
I don’t think most users care about commit numbers. 
Do you think this will serve a good purpose? Instead we can just tell them “run this command for the most up-to-date list”.